RSSReader/Presenter/FeedPresenter.m

```diff
@@ -0,0 +1,63 @@
+//
+//  FeedPresenter.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 19.11.20.
+//
+
+#import "FeedPresenter.h"
+#import "NetworkManager.h"
+#import "FeedItem.h"
+#import "RSSParser.h"
+
+@interface FeedPresenter ()
+
+@property (nonatomic, retain) NetworkManager *networkManager;
+@property (nonatomic, retain) RSSParser *parser;
+@property (nonatomic, retain) NSMutableArray* feedItemArray;
+
+@end
+
+@implementation FeedPresenter
+
+- (id)initWithArray:(NSMutableArray *)array networkManager:(NetworkManager *)manager parser:(RSSParser *)parser{
+    if (self = [super init]) {
+        if (array) {
+            _feedItemArray = array;
+            [_feedItemArray retain];
+        }
+        else {
+            _feedItemArray = [NSMutableArray new];
+        }
+        _networkManager = manager;
+        [_networkManager retain];
```

Comment:
look through the documentation, the `retain` method returns `instancetype` with references count + 1
you should write _networkManager = [manager retain]

Reply:
fixed